### PR TITLE
Add support of test network for collectables.

### LINKED ISSUE #747: Rinkeby and OpenSea.io
When using Rinkeby network and viewing an individual collectible (ERC721) and pressing Open on OpenSea.io this opens https://opensea.io/assets/[contract]/[tokenID] when it should open https://rinkeby.opensea.io/assets/[contract]/[tokenID]

Not sure if this should be fixed on the trust side or opensea.

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
Add option for test network to browse collectables.

Fix #747

### CODE REVIEW

File: Trust/Extensions/URL.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Copyright SIX DAY LLC. All rights reserved.
+
+import UIKit
+
+extension URL {
+
+    private static var openseaPath: String {
```
Comment (vikmeup) [inline Trust/Extensions/URL.swift]: can you make this part of `RPCServer` or `Config`?

File: Trust/Extensions/URL.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        switch Config().server {
+        case .rinkeby:
```
Comment (vikmeup) [inline Trust/Extensions/URL.swift]: make switches exhaustive

File: Trust/Extensions/URL.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/Extensions/URL.swift]: Done it is nice idea.

File: Trust/Extensions/URL.swift (hunk 2 above)
Comment (OlegGordiichuk) [reply Trust/Extensions/URL.swift]: With enums this is not required.

File: Trust/Settings/Types/RPCServers.swift [hunk 3]
```diff
@@ -134,6 +134,15 @@ enum RPCServer {
         }
     }
 
+    var openseaPath: String {
+        switch self {
```
[the remark this answers is not in the document]
Comment (OlegGordiichuk) [reply Trust/Settings/Types/RPCServers.swift]: With enums we could avoid this.

File: Trust/Settings/Types/Config.swift [hunk 4]
```diff
@@ -66,4 +66,12 @@ struct Config {
         get { return defaults.bool(forKey: Keys.testNetworkWarningOff) }
         set { defaults.set(newValue, forKey: Keys.testNetworkWarningOff) }
     }
+
+    func opensea() -> URL? {
+        return URL(string: server.openseaPath)
```
[the remark this answers is not in the document]
Comment (OlegGordiichuk) [reply Trust/Settings/Types/Config.swift]: Yeah

File: Trust/Settings/Types/RPCServers.swift (hunk 3 above)
[the remark this answers is not in the document]
Comment (OlegGordiichuk) [reply Trust/Settings/Types/RPCServers.swift]: Done

File: Trust/Settings/Types/RPCServers.swift [hunk 5]
```diff
@@ -136,10 +136,15 @@ enum RPCServer {
 
     var openseaPath: String {
         switch self {
-        case .rinkeby:
-            return "https://rinkeby.opensea.io"
-        default:
-            return Constants.dappsOpenSea
+        case .main: return  Constants.dappsOpenSea
```
Comment (vikmeup) [inline Trust/Settings/Types/RPCServers.swift]: Do case .poa, .kovan, ....

Comment (OlegGordiichuk) [reply Trust/Settings/Types/RPCServers.swift]: Done

**Pull request merged**